include/fiction/algorithms/physical_design/apply_gate_library.hpp

```diff
@@ -172,7 +172,8 @@ class apply_gate_library_impl
                 // "copy_lyt". When using "cell_lyt.assign_sidb_defect(...)" inside the lambda function, it results in
                 // the error: "error C2059: syntax error: '.'".
                 auto copy_lyt = cell_lyt.clone();
-                // add defects to the circuit.
+                // copy the original defects over to the circuit since they are gone when converting the gate-level
+                // layout to the cell-level layout.
                 lyt.value().foreach_sidb_defect([this, &copy_lyt](const auto& def)
```

Comment:
Does this code address the age-old `TODO` in the defect-aware physical design experiment? https://github.com/cda-tum/fiction/blob/main/experiments/defect_aware_physical_design/defect_aware_physical_design.cpp#L200